core/main/src/processor/main_context_processor.rs

```diff
@@ -199,7 +199,20 @@ impl MainContextProcessor {
             if state.supports_app_catalog() {
                 let ignore_list = vec![state.get_device_manifest().applications.defaults.main];
                 let client = state.get_client();
-                client.add_event_processor(AppsUpdater::new(client.get_extn_client(), ignore_list));
+                let uninstalls_enabled = RemoteFeature::flag(
```

Comment:
Uses the new RemoteFeature impl code to fetch a uninstalled_enabled flag on boot and then pass that to the AppsUpdater module.

Reply:
This manifest design patten is very scalable. We can probably work with Accelerator UI folks to define some for the VA Manifests. cc @arun-madhavan-013 @joseinweb